model/model_training/configs/config.yaml

```diff
@@ -265,12 +275,17 @@ pythia-12B:
   learning_rate: 6e-6
   model_name: EleutherAI/pythia-12b-deduped
   weight_decay: 0.0
-  max_length: 2048
-  warmup_steps: 20
+  max_length: 620
```

Comment:
Is this right? I thought we have been doing SFT runs with 2048?

Reply:
yes, you're right! was confused by a previous config